package.json

```diff
@@ -21,42 +21,40 @@
   },
   "scripts": {
     "build": "yarn build:components",
-    "build:all": "run-s build:preview build:site",
-    "postbuild:all": "yarn mv:preview",
-    "build:clean": "run-s clean build",
-    "build:components": "lerna run --scope \"${SCOPE:-@spectrum-css/*}\" --ignore \"@spectrum-css/{*-builder*,preview,generator}\" build",
-    "postbuild:components": "yarn docs:mod",
-    "build:preview": "nx build @spectrum-css/preview",
-    "build:site": "gulp buildDocs -LLL",
-    "cache:clean": "nx reset",
+    "build:all": "run-s build:preview build:site mv:preview",
+    "build:components": "nx run-many --all --exclude \"@spectrum-css/{*-builder*,preview,generator}\" --target build",
+    "postbuild:components": "node tasks/mod-extractor.js && prettier --cache --write components/*/metadata/*.md > /dev/null",
+    "build:preview": "nx run @spectrum-css/preview:build",
+    "build:site": "gulp build -LLL",
     "ci": "yarn build:all",
-    "ci:storybook": "nx storybook @spectrum-css/preview",
-    "clean": "yarn cache:clean && run-p clean:*",
-    "clean:components": "lerna exec --scope \"${SCOPE:-@spectrum-css/*}\" --ignore \"@spectrum-css/{*-builder*,preview,generator}\" -- rimraf dist",
-    "clean:docs": "rimraf dist",
-    "clean:preview": "nx clean @spectrum-css/preview",
-    "predev": "yarn build:components",
-    "dev": "NODE_ENV=development BROWSERSYNC_OPEN=true gulp dev",
-    "docs:mod": "node tasks/mod-extractor.js",
-    "postdocs:mod": "prettier --cache --write components/*/metadata/*.md > /dev/null",
-    "lint:components": "node ./tasks/packageLint.js",
-    "lint:styles": "stylelint",
-    "mv:preview": "rimraf dist/preview && mv tools/preview/storybook-static dist/preview",
+    "ci:storybook": "nx run @spectrum-css/preview:storybook",
+    "clean": "run-p clean:*",
```

Comment:
I had about 90% success with running this command, but there were a few occasions when it seemingly hung for a few moments and then failed. It might have gotten tripped up on cleaning the Preview stuff? I'm unsure, and it seemed a little fluke-y.

<img width="752" alt="Screenshot 2023-08-22 at 1 29 37 PM" src="https://github.com/adobe/spectrum-css/assets/360251/4bd382a6-efae-4517-b5cb-a6f4a0beade4">

Reply:
Oh hmm, this looks related to the cache reset happening concurrent to the other commands, maybe if it's run first

Reply:
This should now be resolved 🥳